src/app/frontend/overview/overview.html

```diff
@@ -29,6 +29,75 @@
   </kd-graph-card>
 </div>
 
+<div layout="row" layout-align="center" ng-if="$ctrl.podStats.total">
+    <div class="kd-graph-container">
+        <kd-allocated-resources-chart outer="$ctrl.podStats.pendingPodsFraction"
+                                      outercolor="$ctrl.pendingColor">
+        </kd-allocated-resources-chart>
+        <div class="kd-graph-legend">
+          <div class="kd-graph-legend-title">
+            Pending Pods
+          </div>
+          <div class="kd-graph-legend-entry">
+            <md-icon class="kd-graph-yellow">stop</md-icon>
+            [[Pending|Label 'Pending' for the pod statistics graphs legend.]]
```

Comment:
@maciaszczykm  is this content trailing error causing travis fail related to single quotes in description?